Doc/whatsnew/3.15.rst

```diff
@@ -318,6 +318,11 @@ ssl
 
    (Contributed by Ron Frederick in :gh:`136306`)
 
+* Added new method :meth:`ssl.SSLContext.set_ciphersuites` for setting TLS 1.3
+  ciphers and updated the documentation on :meth:`ssl.SSLContext.set_ciphers`
+  to mention that it only applies to TLS 1.2 and earlier and that this new
+  method must be used to set TLS 1.3 cipher suites.
```

Comment:
```suggestion
* Added new method :meth:`ssl.SSLContext.set_ciphersuites` for setting TLS 1.3
  ciphers. For TLS 1.2 or earlier, use :meth:`ssl.SSLContext.set_ciphers` instead.
```

Reply:
What do you think about changing the text here to something like:

```
* Added new method :meth:`ssl.SSLContext.set_ciphersuites` for setting TLS 1.3
  ciphers. For TLS 1.2 or earlier, :meth:`ssl.SSLContext.set_ciphers` should
  continue to be used. Both calls can be made on the same context and the
  selected cipher suite will depend on the TLS version negotiated when a
  connection is made.
```

I wanted to somehow capture that these calls aren't mutually exclusive if you don't know in advance what version of TLS will end up being used. The existing call only affects cipher suites chosen when TLS 1.2 or earlier is negotiated, and the new call only affects cipher suites chosen when TLS 1.3 is negotiated.

Reply:
@picnixz, are you ok with this new wording?

Reply:
I've checked in the new wording I proposed. Let me know if you want any changes here.

I think this change may be ready to go if there are no other review comments.

Reply:
Sorry, I forgot about this PR and I missed the notification I think. Let me have a look again.